docs/dive-in/2024/01/year-month-day:name-of-feature.md

```diff
@@ -0,0 +1,29 @@
+# TITLE: You know what you’re working, so in a few words give us a clear title
+
+Author : John Doe
+Date : 2024-01-01
+
+## SUMMARY:
+
+Gather all the information you have grasped and start by writing a
+quick summary of what you want to do and why you’re doing it in one line or two.
+
+## CONTEXT:
+
+Share with them, we need to know the situation your in to understand the summary. Give the juicy details of your project and show us why it’s important
+to have: the new feature, this new technology, this language, that database…etc.
+This should have a minimum of one paragraph and more if needed, share the value
+of the it #AGILE.
+
+## SOLUTION:
+
+What everyone is waiting for, how you’re going to resolve your problem.
+Write details, have a scheme, pseudo-code, poc, a step-by-step processs if
+needed.
+
+Make them understand how you’re going to smash this project with the critical
+information they need.
+
+You will be challenged (normally) and reviewed so make sure you know your
+main cases and all possible edge cases, there might be angles that you have
+not seen but that’s what the review is for 😉
```

Comment:
Would be nice if template files are either put in a path containing the word `template` or their filename contains it, or at it inside them it is mentioned that it's a template.